skipper.go

```diff
@@ -802,6 +811,20 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 		}
 	}
 
+	if o.RemoteRoutesFile != "" {
+		for _, rf := range strings.Split(o.RemoteRoutesFile, ",") {
+			client, err := eskipfile.RemoteWatch(&eskipfile.RemoteWatchOptions{
+				RemoteFile:    rf,
+				FailOnStartup: o.RemoteRoutesFailOnStartup,
+			})
+			if err != nil {
+				log.Error("error while loading remote eskip file", err)
```

Comment:
lets include failed url - in case skipper is started with multiple arguments it would be easier to debug which url was not available

Reply:
Though maybe `err` would include it, still lets have a look from user perspective debugging the startup failure